extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -148,6 +157,13 @@ define('ext.wikia.adEngine.template.porvata', [
 
 		log(['show', params], log.levels.debug, logGroup);
 
+		if (!isVideoAutoplaySupported()) {
+			log(['hop', params.adProduct, params.slotName, params], log.levels.info, logGroup);
+			slotRegistry.get(params.slotName).hop();
```

Comment:
Can we pass object with informations to track to `hop` method? Like here:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/provider/gpt/adDetect.js#L199
For example:
```
{
  adType: 'hop',
  passback: 'porvata'
}
```